src/UseCases/DataAccessInterfaces/VendorRepository.java

```diff
@@ -53,4 +53,6 @@
     Boolean isTokenValid(String token);
 
     boolean save(IVendor vendor);
+
+    RegularVendor getUserFromToken(String token);
```

Comment:
Why do you need getUser methods in the vendor repository? If you wanted to access the user objects wouldn't you just call the methods in userRepository?

Reply:
Oh right, I didn't think of that. I'll remove that, change the other file that uses that method, and have those changes in my next pull request.